QStreak/View Controllers/Record List/RecordListViewModel.swift

```diff
@@ -13,23 +13,52 @@ protocol RecordListViewModelDelegate: AnyObject {
 }
 
 class RecordListViewModel {
-    
     // MARK: - Properties
 
+    var records: [Submission?] {
+        didSet { recordsDidChange?(records) }
+    }
+
+    var recordsDidChange: (([Submission?]) -> Void)?
+    var currentPage = 1
+    var totalPages = 2
```

Comment:
Why is this `2`?

Reply:
So that we can get past the guard clause in `fetchRecords`. Actually, it probably makes the most sense for the initial value to be `1`. That'll get past the guard clause on the first request and the real value will be set later on in `fetchRecords`